src/Stratis.Bitcoin.Features.Consensus/StakeValidator.cs

```diff
@@ -49,9 +49,6 @@ public class StakeValidator : IStakeValidator
         /// <summary>When checking the POS block signature this determines the maximum push data (public key) size following the OP_RETURN in the nonspendable output.</summary>
         private const int MaxPushDataSize = 40;
 
-        /// <summary>Expected (or target) block time in seconds.</summary>
-        public const uint TargetSpacingSeconds = 64;
```

Comment:
I'd actually put this property in Network instead of options

Reply:
Shoudl there still be a differentiation between POW and POS target spacing? StratisMain did have  POW era?